src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs

```diff
@@ -5,61 +5,167 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
-
 namespace System.IO.IsolatedStorage
 {
+    public partial interface INormalizeForIsolatedStorage
+    {
+        object Normalize();
+    }
+
+    public abstract partial class IsolatedStorage // : System.MarshalByRefObject
+    {
+        protected IsolatedStorage() { }
+        public object ApplicationIdentity { get { throw null; } }
+        public object AssemblyIdentity { get { throw null; } }
+        public virtual long AvailableFreeSpace { get { throw null; } }
+        [System.CLSCompliantAttribute(false)]
+        [System.ObsoleteAttribute("IsolatedStorage.CurrentSize has been deprecated because it is not CLS Compliant.  To get the current size use IsolatedStorage.UsedSize")]
+        public virtual ulong CurrentSize { get { throw null; } }
+        public object DomainIdentity { get { throw null; } }
+        [System.CLSCompliantAttribute(false)]
+        [System.ObsoleteAttribute("IsolatedStorage.MaximumSize has been deprecated because it is not CLS Compliant.  To get the maximum size use IsolatedStorage.Quota")]
+        public virtual ulong MaximumSize { get { throw null; } }
+        public virtual long Quota { get { throw null; } }
+        public System.IO.IsolatedStorage.IsolatedStorageScope Scope { get { throw null; } }
+        protected virtual char SeparatorExternal { get { throw null; } }
+        protected virtual char SeparatorInternal { get { throw null; } }
+        public virtual long UsedSize { get { throw null; } }
+        // protected abstract System.Security.Permissions.IsolatedStoragePermission GetPermission(System.Security.PermissionSet ps);
```

Comment:
PermissionSet was added yesterday to dev/API, so you could pay down this debt now.  It would require that you depend on netstandard1.7 instead.

If you want to just file a tracking bug, that would be fine.